dockernet/dockerfiles/Dockerfile.umee

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:22.04
```

Comment:
replace this with

```dockerfile
FROM golang:1.19-alpine3.15 AS builder

WORKDIR /opt

RUN apk add --update ca-certificates build-base git

ENV COMMIT_HASH=d50edeea01c34ac342b6d3f505e29836ccd3a381

RUN git clone https://github.com/umee-network/umee.git \
    && cd umee \
    && git checkout $COMMIT_HASH 

WORKDIR /opt/umee

ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0/libwasmvm_muslc.aarch64.a /lib/libwasmvm_muslc.aarch64.a
ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0/libwasmvm_muslc.x86_64.a /lib/libwasmvm_muslc.x86_64.a
RUN sha256sum /lib/libwasmvm_muslc.aarch64.a | grep 7d2239e9f25e96d0d4daba982ce92367aacf0cbd95d2facb8442268f2b1cc1fc
RUN sha256sum /lib/libwasmvm_muslc.x86_64.a | grep f6282df732a13dec836cda1f399dd874b1e3163504dbd9607c6af915b2740479
RUN cp /lib/libwasmvm_muslc.$(uname -m).a /lib/libwasmvm_muslc.a

RUN LEDGER_ENABLED=false BUILD_TAGS=muslc LINK_STATICALLY=true make build

FROM alpine:3.15
COPY --from=builder /opt/umee/build/umeed /usr/local/bin/
RUN apk add --update bash vim ca-certificates \
    && addgroup -g 1000 umee \
    && adduser -S -h /home/umee -D umee -u 1000 -G umee

USER 1000
WORKDIR /home/umee

EXPOSE 26657 26656 1317 9090

CMD ["umeed", "start"]
```

Reply:
@sampocs , ```**umeed start**``` won't be enough ! That is why we had the node_init script to bootstrap the node. 

Under the expose, we need to add

```
RUN umeed init "umee-1" --chain-id umee-chain
RUN umeed keys add ACC_1 --keyring-backend test
RUN umeed add-genesis-account ACC_1 10000000000000000000000uumee --keyring-backend test
RUN umeed gentx ACC_1 1000000uumee --chain-id umee-chain --keyring-backend test
RUN umeed collect-gentxs
CMD ["sh", "-c", "umeed start --minimum-gas-prices 0.1uumee"]```

Reply:
Now its running and producing blocks.

```0323438203134322031393520323439203132332031373120323231203730203138372031363320313137203138342031373520313720313132203137312033332036305D3A327D
11:14AM INF committed state app_hash=B3435758F8EAD0E77BFF488F7ACAF88EC3F97BABDD46BBA375B8AF1170AB213C height=2 module=state num_txs=0
11:14AM INF indexed block exents height=2 module=txindex
11:14AM INF Timed out dur=4978.125116 height=3 module=consensus round=0 step=1
11:14AM INF received proposal module=consensus proposal={"Type":32,"block_id":{"hash":"EBFE5122A329E7429C24C6D648C0B38539BF3C570643A163C7E6EE1A71D32ED5","parts":{"hash":"990FCB20E129069CA75FD21EFFBFD3F1E05CFDDA6D2DA3A8ACB73095EA085460","total":1}},"height":3,"pol_round":-1,"round":0,"signature":"0DE6ge0oRB/xxjECNwIpUy/daocsd/ldMdHL5OFWWWF0yFbgHZrH18LDUbADmP2YMRHrUcuVQe12D9jHUP/6DQ==","timestamp":"2023-05-01T11:14:40.83539658Z"}
11:14AM INF received complete proposal block hash=EBFE5122A329E7429C24C6D648C0B38539BF3C570643A163C7E6EE1A71D32ED5 height=3 module=consensus
11:14AM INF finalizing commit of block hash={} height=3 module=consensus num_txs=0 root=B3435758F8EAD0E77BFF488F7ACAF88EC3F97BABDD46BBA375B8AF1170AB213C
11:14AM INF minted coins from module account amount=205972677179287uumee from=mint module=x/bank
11:14AM INF executed block height=3 module=state num_invalid_txs=0 num_valid_txs=0
11:14AM INF commit synced commit=436F6D6D697449447B5B3138322032343020313936203935203135362031303820363520313820323330203233382034372031323820313839203131372037332033322032322031303320323534203138312031352032343820313236203134342032343320313037203138302031373020313330203139382033332035355D3A337D
11:14AM INF committed state app_hash=B6F0C45F9C6C4112E6EE2F80BD7549201667FEB50FF87E90F36BB4AA82C62137 height=3 module=state num_txs=0
11:14AM INF indexed block exents height=3 module=txindex```